Molio/Source/Data/DataSource/Network/FirebaseService/DefaultFirebaseAuthService.swift

```diff
@@ -0,0 +1,49 @@
+import FirebaseAuth
+
+struct DefaultFirebaseAuthService: AuthService {
+    
+    func getCurrentUser() throws -> User? {
```

Comment:
P5;
```suggestion
    func getCurrentUser() throws -> User {
```

현재 유저가 없는 경우 `throw`를 하기에 `User?` 말고 `User` 를 반환해도 될 것 같습니다.

Reply:
넵 반영하겠습니다!